components/aws/sagemaker/unit_tests/README.md

```diff
@@ -0,0 +1,25 @@
+# Unit tests for AWS Sagemaker KFP Components 
```

Comment:
**Please** capitalise the `M` in SageMaker for all documentation.

Reply:
done